tests/layer1/test_port_error.py

```diff
@@ -44,23 +49,44 @@ def get_mac_fault_count(dut, interface, fault_type):
     def get_interface_status(dut, interface):
         return dut.show_and_parse("show interfaces status {}".format(interface))[0].get("oper", "unknown")
 
-    @pytest.fixture
+    # TODO: Reboot the switch before test as a WA of #22205, remove this after the issue is fixed
```

Comment:
@gshemesh2 please don't introduce this workaround of reboot. User should prepare the testbed for the desired HWSKU under test and the testbed preparation is outside the scope of this test